.github/workflows/configlet.yml

```diff
@@ -16,3 +16,6 @@ jobs:
 
       - name: Configlet Linter
         run: configlet lint
+
+      - name: Ensure valid JSON
+        run: jq '.' config.json
```

Comment:
This is actually a very nice idea. @ee7 @saschamann do we want to add this to the central configlet workflow to catch https://github.com/exercism/configlet/issues/312?

Reply:
Great idea. Why didn't we think of this before?

Created a draft PR here: https://github.com/exercism/github-actions/pull/21

Maybe we can make this look obvious in the workflow log, rather than needing to do a mass PR to each track's `configlet.yml`.

Reply:
> But I've just realised that we can just make configlet lint run jq when it's detected on the system (and/or when we're running in CI). Let's do that instead.

Now we're thinking! :-)  It could even recommend you install it...